.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -1,13 +1,17 @@
 <!-- Thanks for contributing to _OHHTTPStubs_! Before you submit your pull request, please make sure to check the following boxes by putting an x in the [ ] -->
```

Comment:
I just realised that HTML comments like those, not being markdown, won't be rendered, so that means that using underscores here isn't really useful. Maybe remove the underscores around `OHHTTPStubs` to make it read better when unformatted? (same for the issue template obviously)